pkg/sqldb/postgres_test.go

```diff
@@ -24,16 +24,14 @@ func TestFetchPhysicalAssetsErrors(t *testing.T) {
 		DatabaseName: "name",
 	}
 
-	if err := thedb.Init(context.Background(), postgresConfig.Hostname, postgresConfig.Port, postgresConfig.Username, postgresConfig.Password, postgresConfig.DatabaseName); err == nil {
-		t.Errorf("DB.Init should have returned a non-nil error")
-	}
+	require.Error(t, thedb.Init(context.Background(), postgresConfig.Hostname, postgresConfig.Port,
+		postgresConfig.Username, postgresConfig.Password, postgresConfig.DatabaseName),
+		"DB.Init should have returned a non-nil error")
 }
 
 func TestDoesDBExistTrue(t *testing.T) {
 	mockdb, mock, err := sqlmock.New()
-	if err != nil {
-		t.Fatalf("an error '%s' was not expected when opening a stub database connection", err)
-	}
+	require.Nil(t, err, "an error '%s' was not expected when opening a stub database connection")
```

Comment:
nit: to take advantage of string interpolation with `require.Nil`, you have to pass the arguments to you wish to interpolate into the string to the `Nil` function. Example: https://play.golang.org/p/HqTYpnKFpmH

Reply:
🤦‍♂ will fix